documents/operating.md

```diff
@@ -52,14 +52,202 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
 SECTION 5: 🗣🗣🗣 Governing Process 🗣🗣🗣
 ---------------------------------------------------------------------
-- [ ] Types of decision
-- [ ] Records and documentation
-- [ ] Membership
-- [ ] Mechanisms for deliberation
-  - [ ] ... On GitHub
-  - [ ] ... On Slack
-- [ ] Use of emoji
-- [ ] Meetings
+- [x] Types of decision
+- [x] Records and documentation
+- [x] Mechanisms for deliberation
+  - [x] ... On GitHub
+  - [x] ... On Slack
+  - [x] ... Plotting Sessions
+- [x] Use of emoji
+
```

Comment:
Should add another section for elections specifically -- how are elections held / what is winning / how are ties broken / etc.